core/src/test/scala/org/apache/spark/rdd/RDDSuite.scala

```diff
@@ -656,7 +656,7 @@ class RDDSuite extends SparkFunSuite with SharedSparkContext with Eventually {
   }
 
   test("top with predefined ordering") {
-    val nums = Array.range(1, 100000)
+    val nums = Seq.range(1, 100000)
```

Comment:
Side comment: generally speaking Seq types have less weird generic type problems than Arrays. This is a good example